com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -553,6 +573,11 @@ var GameArchiveLoader = {
         ++file.totalLoadedPieces;
         // is all pieces of the file loaded?
         if (file.totalLoadedPieces == file.pieces.length) {
+            if (file.stream !== undefined) {
+                FS.close(file.stream);
+                file.stream = undefined;
+            }
+            this.verifyFile(file);
```

Comment:
What's the rationale for moving the function call?

Reply:
This moved because onFileDownloaded is called in two places now. One when the parts are downloaded and can be verified, the other were the parts have already been put back into a single file.